src/components/BetForm/BetForm.module.scss

```diff
@@ -0,0 +1,88 @@
+@import "../../ui/Colors.module.scss";
```

Comment:
you can try in Vite make preload this scss file and it will no need to import it in all files